flyway-core/src/main/java/org/flywaydb/core/internal/util/VersionPrinter.java

```diff
@@ -42,6 +42,6 @@ public static void printVersion(ClassLoader classLoader) {
         }
         printed = true;
         String version = new ClassPathResource("org/flywaydb/core/internal/version.txt", classLoader).loadAsString("UTF-8");
-        LOG.info("Flyway " + version + " by Boxfuse");
+        LOG.info("Flyway " + version + " by Boxfuse XXX");
```

Comment:
No. Removed in https://github.com/rollbar/flyway/commit/a32624a23d552f770aa8cb2132c8be09042c3064